doc_user/UserManual.adoc

```diff
@@ -2263,349 +2263,342 @@ Once you have pressed the *[.underline]##T##ypes* button on the *_Settings_* men
 
-Once you have pressed the *Delive[.underline]##r##y Result Type* button from the *_Types_* menu you access the *_Delivery Result Type Browser_* window as shown below:
+Once the *Delive[.underline]##r##y Result Type* button from the *_Types_* is pressed, the *_Delivery Result Type Browser_* window shown below appears:
 
-image:image143.png[Discharge Types.PNG,width=432,height=194]
+image:image143.png[Discharge Result Types,width=453,height=203]
 
 The functions available in this window (similar to other windows in the *_Types_* menu) are:
 
-* *[.underline]##N##ew*: to enter a new type (cannot use an already existing code)
-* *[.underline]##E##dit*: to edit an already existing type (the code must be unchanged)
-* *[.underline]##D##elete*: to delete an already existing type (cannot be deleted if still used somewhere in the application)
+* *[.underline]##N##ew*: to enter a new type (cannot use an existing code)
+* *[.underline]##E##dit*: to edit an existing type (the code cannot be changed)
+* *[.underline]##D##elete*: to delete an existing type (cannot be deleted if still used somewhere in the application)
 * *[.underline]##C##lose*: to close the window
 
-The Delivery Result Types defined in this functionality will be reflected in the Admission function (see
+The Delivery Result Types defined in this functionality are used in the Admission function (see
 <<insert-or-modify-delivery-data,Insert or Modify Delivery data>> in this document).
 
 ====  14.2.5 Disease Types (Disease [.underline]##T##ype)
 
-Once you have pressed the *Disease [.underline]##T##ype* button from the *_Types_* menu you access the *_Disease Type Browser_* window as shown below:
+Once the *Disease [.underline]##T##ype* button from the *_Types_* menu is pressed, the *_Disease Type Browser_* window shown below appears:
 
-____
-image:image144.PNG[image,width=408,height=183]
-____
+image:image144.PNG[Disease Type,width=453,height=203]
 
 The functions available in this window (similar to other windows in the *_Types_* menu) are:
 
-* *[.underline]##N##ew*: to enter a new type (cannot use an already existing code)
-* *[.underline]##E##dit*: to edit an already existing type (the code must be unchanged)
-* *[.underline]##D##elete*: to delete an already existing type (cannot be deleted if still used somewhere in the application)
+* *[.underline]##N##ew*: to enter a new type (cannot use an existing code)
+* *[.underline]##E##dit*: to edit an existing type (the code cannot be changed)
+* *[.underline]##D##elete*: to delete an existing type (cannot be deleted if still used somewhere in the application)
 * *[.underline]##C##lose*: to close the window
 
-The Disease Types defined in this functionality will be reflected in the entire application, like in Disease definition or in OPD (see
+The Disease Types defined in this functionality are used throughout the application.  For example, in definition of Diseases or in OPD (see
```

Comment:
double space here

Reply:
It doesn't make any difference; the render removes extra spaces:

![image](https://user-images.githubusercontent.com/1105445/124026992-594b7e00-d9c0-11eb-8656-97f2b8c27f5d.png)

Reply:
it makes difference in size? 😁

Reply:
Changed.  You missed the other 8 occurrences of multiple spaces at the end and the start of the next sentence.

Reply:
Thanks!